cli/src/pcluster/aws/cfn.py

```diff
@@ -134,10 +125,18 @@ def describe_stack_resource(self, stack_name: str, logic_resource_id: str):
             )
 
     @AWSExceptionHandler.handle_client_exception
-    def get_imagebuilder_stacks(self):
+    def get_imagebuilder_stacks(self, next_token=None):
         """List existing imagebuilder stacks."""
+        return self._list_pcluster_stacks_with_tag(PCLUSTER_IMAGE_ID_TAG, next_token)
+
+    def _list_pcluster_stacks_with_tag(self, tag, next_token=None):
```

Comment:
minor - what about `_list_stacks_with_tag` or `_list_cfn_stacks_with_tag`?

Reply:
What about `_list_parentless_stacks_with_tag`? It would be more descriptive of what the method actually does

Reply:
ok